email_templates/dos_opportunities.html

```diff
@@ -0,0 +1,711 @@
+
+
+
+                        <img alt="" src="https://gallery.mailchimp.com/2f3ed6987c26fa8f18b3cdd5d/images/36cd6bbc-d08f-4df1-9f7d-ab1da4b2c1c5.jpg" width="264" style="max-width:928px;" class="mcnImage">
+
+
+
+                    </td>
+                </tr>
+            </tbody></table>
+            <table class="mcnCaptionRightTextContentContainer" align="right" border="0" cellpadding="0" cellspacing="0" width="264">
+                <tbody><tr>
+                    <td valign="top" class="mcnTextContent">
+                        <div style="text-align: right;"><br>
+<a href="*|ARCHIVE|*" target="_blank">View this email in your browser</a></div>
+
+                    </td>
+                </tr>
+            </tbody></table>
+        </td>
+    </tr>
+</tbody></table>
+
+
+
+
+            </td>
+        </tr>
+    </tbody>
+</table></td>
+                            </tr>
+                            <tr>
+                                <td valign="top" id="templateHeader"><table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnTextBlock" style="min-width:100%;">
+    <tbody class="mcnTextBlockOuter">
+        <tr>
+            <td valign="top" class="mcnTextBlockInner" style="padding-top:9px;">
+              	<!--[if mso]>
+				<table align="left" border="0" cellspacing="0" cellpadding="0" width="100%" style="width:100%;">
+				<tr>
+				<![endif]-->
+
+				<!--[if mso]>
+				<td valign="top" width="600" style="width:600px;">
+				<![endif]-->
+                <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
+                    <tbody><tr>
+
+                        <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
+
+                            <p><span style="color:#000000"><span style="font-size:19px">Dear supplier,</span></span><br>
+<br>
+<span id="opening-paragraph" style="color:#000000"><span style="font-size:19px">{% if number_of_days == 1 %}In the last day{% else %}Since {{ start_date.strftime("%A") }}{% endif %}, {{ briefs|length }} new {{ briefs[0].lotName|lower }} {% if briefs|length == 1 %}opportunity was{% else %}opportunities were{% endif %} published on the Digital Marketplace. View and apply for {% if briefs|length == 1 %}this opportunity{% else %}these opportunities{% endif %}:</span></span><br>
+<br>
+
+{% for brief in briefs %}
+	<span class="opportunity-title" style="font-size:16px"><font color="#000000"><strong>{{ brief.title }}</strong></font></span><br>
+	<span class="opportunity-organisation" style="color:#000000">{{ brief.organisation }}<br></span>
+	<span class="opportunity-location" style="color:#000000">{{ brief.location }}</span><br>
+	<span class="opportunity-closing" style="color:#000000">Closing {{ (datetime.strptime(brief.applicationsClosedAt, datetime_format)).strftime(display_date_format) }}</span><br>
```

Comment:
Would rather see this date formatting done in `get_html_content` and passed in to the template as template variables; passing the whole library into the template renderer doesn't feel right. The smarter templates are the harder they are to debug/ maintain and it creates a barrier for entry meaning that devs that only know HTML can't maintain your templates for you.